src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa.cs

```diff
@@ -30,9 +30,10 @@ public void InvalidAccessThrows()
         public void InvalidFileModeAccessReadThrows()
         {
             FileMode[] invalidModes = { FileMode.Append, FileMode.Create, FileMode.CreateNew, FileMode.Truncate };
+            int i = 0;
             foreach (FileMode invalidMode in invalidModes)
             {
-                Assert.Throws<ArgumentException>(() => CreateFileStream(GetTestFilePath(), invalidMode, FileAccess.Read));
+                Assert.Throws<ArgumentException>(() => CreateFileStream(GetTestFilePath(i++), invalidMode, FileAccess.Read));
```

Comment:
I read it thinking it was an Assert.All and I didn't want one failed iteration to successfully create the file and cause subsequent failures for the wrong reason. 

Since it is a foreach, you are correct that it isn't a necessary change. I'll roll it back.